test/test_excerpt.rb

```diff
@@ -1,22 +1,23 @@
-require 'helper'
+require "helper"
 
 class TestExcerpt < JekyllUnitTest
   def setup_post(file)
-    Document.new(@site.in_source_dir(File.join('_posts', file)), {
-      site:       @site,
-      collection: @site.posts
+    Document.new(@site.in_source_dir(File.join("_posts", file)), {
+      :site => @site,
+      :collection => @site.posts
```

Comment:
Please align the `@site`'s here. I applied that in 2caff75. Please rebase on master. 😄